docs/_templates/autosummary/class.rst

```diff
@@ -1,5 +1,3 @@
-:github_url: {{ fullname | github_url }}
```

Comment:
Hmm, this is a regression in my eyes. The `[source]` links now link to `_modules/` URLs. I preferred them linking to GitHub. Can this be done?

Reply:
maybe this?

https://www.sphinx-doc.org/en/master/usage/extensions/linkcode.html

Reply:
it's been added